functions_lab.js

```diff
@@ -0,0 +1,418 @@
+// # Function Exercises
+//
+// ## Question 1
+//
+// Write a function that returns the largest of two numbers. If they are equal, return the second one.
+//
+// Example:
+//
+// Input: `12, 4`
+//
+// Output: `12`
+
+console.log('=======answer#1=========')
+
+let num1 = 12;
+let num2 = 4;
+function largestOfTwoNum (num1, num2) {
+  if (num1 < num2) {
+  return num2
+  } else if (num1 > num2) {
+    return num1
+  } else {
+    return num1,num2
```

Comment:
You can't have two values in return. It will only return which ever is the later value. Ex: return num1, num2 => it will always return num2 but not num1